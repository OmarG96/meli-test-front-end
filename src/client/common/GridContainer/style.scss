.grid-container-root {
  padding: 0 5%;
  display: grid;
  grid-template-columns: repeat(12, auto);
  grid-gap: 16px;
}
