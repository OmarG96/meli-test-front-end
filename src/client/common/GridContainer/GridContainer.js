import React from "react";
import { node } from "prop-types";

import "./style.scss";

const GridContainer = ({ children }) => {
  return <div className="grid-container-root">{children}</div>;
};

GridContainer.propTypes = {
  children: node,
};

export default GridContainer;
