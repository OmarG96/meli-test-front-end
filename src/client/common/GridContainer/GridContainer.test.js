import React from "react";
import GridContainer from ".";
import { render } from "@testing-library/react";

describe("GIVEN GridContainer", () => {
  describe("WHEN is rendered", () => {
    it("THEN match to snapshot", () => {
      const { asFragment } = render(
        <GridContainer>
          <div>example</div>
        </GridContainer>
      );
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
