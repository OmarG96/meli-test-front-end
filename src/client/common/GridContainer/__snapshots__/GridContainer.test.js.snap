// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN GridContainer WHEN is rendered THEN match to snapshot 1`] = `
<DocumentFragment>
  <div
    class="grid-container-root"
  >
    <div>
      example
    </div>
  </div>
</DocumentFragment>
`;
